playback_msg msg
---
uint8 isError
